plugins/bulk-import/src/components/PreviewFile/PreviewPullRequestForm.tsx

```diff
@@ -49,15 +49,11 @@ export const PreviewPullRequestForm = ({
   setPullRequest,
   formErrors,
   setFormErrors,
-  entityOwner,
-  setEntityOwner,
 }: {
   repoId: string;
   repoUrl: string;
   pullRequest: PullRequestPreviewData;
   formErrors: PullRequestPreviewData;
-  entityOwner: string;
-  setEntityOwner: (name: string) => void;
   setPullRequest: (pullRequest: PullRequestPreviewData) => void;
```

Comment:
Same question here, couldn't we simplify the PreviewPullRequestForm when just return one PullRequest here?

```suggestion
  setPullRequest: (pullRequest: PullRequestPreview) => void;
```